### PR TITLE
add eth_syncing details for nitro nodes

### PR DESCRIPTION
**Pull request by mahsamoosavi**
This PR adds details for how one can use `eth_syncing` RPC calls to figure out if their node is synced or not (and debug further if needed).

Preview is [here](https://nitro-docs-ggy72hzp3-offchain-labs.vercel.app/node-running/faq#how-can-i-verify-that-my-node-is-fully-synced).

### CODE REVIEW

File: arbitrum-docs/partials/_troubleshooting-nodes-partial.md [hunk 1]
```diff
@@ -68,15 +68,42 @@
...
 
 ### How can I verify that my node is fully synced? {#how-can-i-verify-that-my-node-is-fully-synced}
-<p>You can make an <code>eth_syncing</code> RPC call to your node. If it returns false, it means that the node is fully synced. If it returns an object of information, it means it is still syncing. Here's how you can interpret that information:</p>
+<p>You can make an <code>eth_syncing</code> RPC call to your node. When a nitro node is fully synced, <code>eth_syncing</code> returns the value <code>false</code> (just like a normal Geth node).</p>
+
+<p>When a nitro node is still syncing, <code>eth_syncing</code> returns a map of values to help understand why the node is not synced. Nitro execution and bottleneck are different from a normal Geth node, so <code>eth_syncing</code> output is unique to nitro. Here, we provide more details:</p>
+
+<p></p>
+
+<p><strong>Introduction: Messages, Batches, and Blocks:</strong></p>
```
Comment (TucksonDev) [inline arbitrum-docs/partials/_troubleshooting-nodes-partial.md:77]: Should we remove the word "Introduction" from here? I wouldn't consider this an introduction, just a description of Messages, Batches and Blocks.

File: arbitrum-docs/partials/_troubleshooting-nodes-partial.md [hunk 2]
```diff
@@ -68,15 +68,42 @@
...
-<ul><li><code>{'lastl1blockNum'}</code> specifies what the node sees as the current L1 state</li>
-<li><code>{'blockNum'}</code> is what's the up-to-date L2 block the node is synced to</li>
-<li><code>{'messageOfLastBlock'}</code> is the message number of that block the node is synced to</li>
-<li><code>{'broadcasterQueuedMessagesPos'}</code> specifies the last message number that the feed has broadcasted</li>
-</ul>
 <p></p>
 
+<p><strong><code>{'eth_syncing'}</code></strong><strong> Fields:</strong></p>
+
+<p>🔉 Note that the exact output for <code>eth_syncing</code> RPC call of an out-of-sync nitro node is not considered an stable API. It is still being actively developed and modified without notice between versions.</p>
```
Comment (TucksonDev) [inline arbitrum-docs/partials/_troubleshooting-nodes-partial.md:87]: Not sure it is possible, but I would use a caution notice for this.
Like the ones used in the [Quickstart](https://developer.arbitrum.io/for-devs/quickstart-solidity-hardhat), (search for Caution)

File: yarn.lock
Comment (TucksonDev) [inline yarn.lock:1]: I think we shouldn't have these yarn files there (`yarn.lock` and `node_modules/.yarn-integrity`).
They might have been generated when running `yarn install` in the root of the project (only website and notion-docs have their own package.json). Can you remove them?

File: arbitrum-docs/partials/_troubleshooting-nodes-partial.md (hunk 1 above)
Comment (mahsamoosavi) [reply arbitrum-docs/partials/_troubleshooting-nodes-partial.md:77]: How about "Messages, Batches and Blocks"?

**Review by TucksonDev — APPROVED** [review]

**Pull request merged**